types/stripe/index.d.ts

```diff
@@ -4194,12 +4196,19 @@ declare namespace Stripe {
             /**
              * The name of the person or business that owns the bank account.
              */
-            account_holder_name?: string;
+            account_holder_name?: string | null;
 
             /**
              * The type of entity that holds the account. This can be either "individual" or "company".
              */
-            account_holder_type?: "individual" | "company";
+            account_holder_type?: "individual" | "company" | null;
+        }
+
+        interface IBankAccountVerifyOptions {
```

Comment:
There was previously no type for the `verirfySource` parameters: https://stripe.com/docs/api/customer_bank_accounts/verify